packages/admin/src/providers/dataProvider.js

```diff
@@ -98,4 +91,25 @@ export default {
 			method: 'DELETE',
 		}).then(({ json }) => ({ data: json }));
 	},
+	upload: (resource, params) => {
+		const { data } = params;
+		const { meta = false, files } = data;
+
+		const formData = new FormData();
+
+		files.map((file, index) => {
+			return formData.append(`files[${index}]`, file);
+		});
```

Comment:
O forEach faz mais sentido aqui
```suggestion
		files.forEach((file, index) => {
			return formData.append(`files[${index}]`, file);
		});
```